### ISSUE TITLE
没有远虑，没有近忧——Github的layout如果又改了？

### ISSUE THREAD

**Issue by EthanLin-TWer** [Yaowenjie/travis-github-chrome-extension#5]
看到你的README说，之前也有同学写过同样的插件，不过因为layout改了就不work了。所以，如果同样的事情发生一次，有没有事情可以进一步自动化呢？
- 自动给wjyao@thoughtworks.com发邮件？
- 渲染前做DOM结构检测？

虽然一个人维护的，短期看来也有维护时间和精力也就不必了。我就刷个提交我会乱说…

**Comment by Yaowenjie** [comment]
@linesh-simplicity Good idea! 其实这个我也有考虑过，前几个相关插件也主要是因为layout原因不能work了，这个issue我要keep着，到时候有好的解决方案可以一试、

**Comment by EthanLin-TWer** [comment]
2016-09-15 中秋Github改版，TW观光团前来围观 :laughing:

**Comment by Yaowenjie** [comment]
@linesh-simplicity 哈哈哈哈哈，现在这布局，感觉那个status加的位置就需要重新考虑了

**Comment by EthanLin-TWer** [comment]
感觉项目布局没有什么变化丫～
👻😨😖😱😳😣😰🙈😶😟
🐯🐶🐺🐕🐅🐱🐷🐆🐾🐮

**Comment by linesh-simplicity** [comment]
👻😨😖😱😳😣😰🙈😶😟

**Comment by EthanLin-TWer** [comment]
卧槽，https://github.com/jasonlong/isometric-contributions 插件又可以 work 了，速度真快。

![image](https://cloud.githubusercontent.com/assets/11895199/18650976/ff6a6e5c-7efa-11e6-870c-23fb88110c92.png)

然而还是觉得这个插件没什么乱用，我要禁用楽

**Comment by Yaowenjie** [comment]
@linesh-simplicity 哈哈哈哈哈哈，为毛我的isometric-contributions 并没有work

**Comment by JimmyLv** [comment]
哈哈哈哈 你们竟然在这闲聊。

**Comment by Yaowenjie** [comment]
@linesh-simplicity welcome!
